oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/record-requests/components/RequestList.jsx

```diff
@@ -16,9 +16,9 @@ import { RequestModal } from ".";
 /**
  * @param {{ requests: Request[], requestModalType: RequestTypeEnum }} props
  */
-export const RequestList = ({ requests, requestTypes, requestModalType }) => {
+export const RequestList = ({ requests, requestTypes, requestModalType, fetchNewRequests }) => {
```

Comment:
FetchNewRequests shouldn't be passed from parent as prop, RequestList component should be responsible for fetching the list of requests